src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,40 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int AGE = 18;
+    private Storage storage = new Storage();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() == null) {
+            throw new RuntimeException("You havnt entered any age");
+        }
+        if (user.getPassword() == null || user.getPassword() == "") {
+            throw new RuntimeException("Password can not be empty");
+        }
+        if (user.getLogin() == null || user.getLogin() == "") {
+            throw new RuntimeException("Login can not be empty");
+        }
+        if (user == null) {
+            throw new RuntimeException("Enter data");
+        }
```

Comment:
I think we need to make this check first(move it above `if (user.getAge() == null) {` check)